astropy/io/votable/tests/data/regression.xml

```diff
@@ -18,7 +18,12 @@ The VOTable format is an XML standard for the interchange of data represented as
   This is a resource description
 </DESCRIPTION>
 <PARAM ID="awesome" datatype="float" name="INPUT" value="0.000000,0.000000" arraysize="*" unit="deg"></PARAM>
-<PARAM ID="empty_value" name="empty_value" arraysize="*" datatype="char" value="" unit="foo"/>
+<PARAM ID="empty_value" arraysize="*" datatype="char" name="empty_value" unit="foo" value="">
+  <VALUES>
+    <OPTION name="empty_value" value=""/>
+    <OPTION name="full_value" value="90prime"/>
```

Comment:
If I remember correctly, the actual data reported in #10572 does not have `name` defined for all the options. So maybe remove `name` from the second option in the regression test data?

```suggestion
    <OPTION value="90prime"/>
```

Reply:
A fair point.  I'll update this and the truth values.